### PR TITLE
Remove legacy policy feedback feature from API

### PR DESCRIPTION
**Pull request by reupen**
### Description of change

This removes nearly all of the old version of the policy feedback feature from the API and the related permissions.

The old acceptance tests were already removed from the front end, so there should be no impact on the acceptance tests (but there is still some clean-up in the front end to do as well).

I haven't explicitly put anything in the migration to check if there are still any any interactions with the `policy_feedback` kind – but can do if we think it is warranted (they have been removed from all deployed environments already).

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [x] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/interaction/models.py [hunk 1]
```diff
@@ -57,13 +57,10 @@ class InteractionPermission(StrEnum):
 
     view_all = 'view_all_interaction'
     view_associated_investmentproject = 'view_associated_investmentproject_interaction'
-    view_policy_feedback = 'view_policy_feedback_interaction'
```
Comment (marcofucci) [inline datahub/interaction/models.py]: There are still some references to these permissions in the docstring that would be good to remove.

File: datahub/interaction/permissions.py [hunk 2]
```diff
@@ -20,19 +17,9 @@ class _PermissionTemplate(StrEnum):
 
     all = '{app_label}.{action}_all_{model_name}'
     associated_investmentproject = '{app_label}.{action}_associated_investmentproject_{model_name}'
-    policy_feedback = '{app_label}.{action}_policy_feedback_{model_name}'
     standard = '{app_label}.{action}_{model_name}'
 
 
-# Mapping from kind to permission required to add/edit/view interactions with that kind
-# (None means no additional permission required over standard interaction permissions.)
-_KIND_PERMISSION_MAPPING = {
```
Comment (marcofucci) [inline datahub/interaction/permissions.py]: There are still some references to this in the docstring `interaction/models.py` that would be good to remove.

**Comment by marcofucci** [comment]
The frontend has some references to `POLICY_FEEDBACK_PERMISSIONS` which would be nice to get rid of.

Also, `setup-uat.sh` still has some references to `policy_feedback_group`, `policy_feedback_staff_user` etc.

File: datahub/interaction/models.py (hunk 1 above)
Comment (reupen) [reply datahub/interaction/models.py]: Thanks, this should be done now.

File: datahub/interaction/permissions.py (hunk 2 above)
Comment (reupen) [reply datahub/interaction/permissions.py]: Thanks, this should be done now.

**Comment by reupen** [comment]
> The frontend has some references to POLICY_FEEDBACK_PERMISSIONS which would be nice to get rid of.
> 
> Also, setup-uat.sh still has some references to policy_feedback_group, policy_feedback_staff_user etc.

Thanks, the front end changes are covered by:

https://github.com/uktrade/data-hub-frontend/pull/1683

We will need that to be merged first, so I will mark this as blocked for the minute.

**Pull request merged**